.appveyor.yml

```diff
@@ -13,6 +13,7 @@ environment:
   STACK_VERSION: 2.1.1
   VIRTUAL_ENV_VERSION: 15.1.0
   YARN_VERSION: 1.16.0
+  RUST_VERSION: 1.35.0
```

Comment:
Nit: We generally try to keep this list sorted alphabetically for easier lookup.